src/test/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/transport/CodecsTest.kt

```diff
@@ -154,6 +154,7 @@ class CodecsTest {
 
                 assertEquals(parsedMessage, decoded, "unexpected parsed result decoded")
                 assertEquals(
+                    //FIXME: org.opentest4j.AssertionFailedError: unexpected raw body ==> expected: <{"field":"16:36:38.035420"}> but was: <{"field":"16:36:38.03542"}>
```

Comment:
We should add a corresponding formatter to each type:

LocalDate: value.toString()
LocalTime: value.format(DateTimeFormatter.ISO_LOCAL_TIME)
LocalDateTime: value.format(DateTimeFormatter.ISO_LOCAL_DATE_TIME)
Instant: value.toString()

Reply:
Do you have example or link to solution?

Reply:
No, I don't have an example or link to the solution. I just checked what is used by Jackson module to serialize those classes and how `toString` method is implemented in those classes

Reply:
You could do something like this:
```kotlin
val testData = listOf<Pair<TemporalAccessor, (TemporalAccessor) -> String>>(
  LocalDate.now() to  value::toString,
  LocalTime.now() to  DateTimeFormatter.ISO_LOCAL_TIME::format,
  LocalDateTime.now() to DateTimeFormatter.ISO_LOCAL_DATE_TIME::format,
  Instant.now() to value::toString,
)